Examples/Nodify.Playground/EditorInputMode.cs

```diff
@@ -0,0 +1,69 @@
+﻿using System.Windows.Input;
+
+namespace Nodify.Playground
+{
+    public enum EditorInputMode
+    {
+        Default,
+        PanOnly,
+        SelectOnly
+    }
+
+    public enum EditorInputMappings
+    {
+        Default,
+        Blender3D
```

Comment:
I suggest using a different name so that others will understand the purpose of this mapping. Maybe `MoveMapping` or `PanMapping`...?

Reply:
Renamed it to Custom